src/app/content/components/popUp/ChapterFilter.tsx

```diff
@@ -102,11 +119,106 @@ const ChapterFilter = ({
   </div>;
 };
 
+// tslint:disable-next-line: variable-name
+export const StyledDetails = styled.details`
+  width: 400px;
+  cursor: pointer;
+  border-bottom: 1px solid ${theme.color.neutral.formBorder};
+  ${theme.breakpoints.mobileSmall(css`
+    width: 100%;
+  `)}
+`;
+
+// tslint:disable-next-line: variable-name
+export const StyledSummary = styled.summary`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 1.6rem;
+  height: 4rem;
+
+  &::marker {
+    content: "";
+  }
+
+  ${ChapterTitle} {
+    margin-left: 0;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const StyledSectionItem = styled(PlainButton)`
+  display: flex;
+  align-items: center;
+  height: 4rem;
+  width: 100%;
+  text-align: left;
+  ${(props: { isSelected: boolean }) => {
+    if (props.isSelected) {
+      return 'color: #027eb5;';
+    }
+  }}
+
+  &:hover {
+    background-color: ${theme.color.neutral.pageBackground};
+  }
+
+  ${ChapterTitle} {
+    padding-left: 2.5rem;
+    width: 95%;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+const ChapterFilterWithToggling = (props: ChapterFilterWithTogglingProps) => {
+  const [isOpenChapterId, setIsOpenChapterId] = React.useState<string | null>(
+    props.selectedSection ? stripIdVersion(props.selectedSection.parent.id) : null
+  );
+
+  return <div className={props.className} tabIndex={-1}>
+    <Row>
+      <Column>
```

Comment:
one column? is that intentional in the design? there was a reason we had to add the column grouping before for large books

Reply:
I think it is scrollable with one column and only one chapter can be expanded
https://projects.invisionapp.com/d/#/console/20557970/433309549/preview

Reply:
hmm... i guess it makes sense because you're gonna have height issues if you expand something in one column and not the other. i would really like to combine these two designs.

maybe if there is any expand/collapse then it uses one column but if there are no entries with children it does the multiple columns

it'll also need a prop for "multiselect" that shows the checkboxes vs the buttons